frontend/src/config/Config.ts

```diff
@@ -124,6 +124,7 @@ const conf = {
       clustersHealth: () => `api/clusters/health`,
       clustersMetrics: () => `api/clusters/metrics`,
       clustersTls: () => `api/clusters/tls`,
+      clustersWorkloads: () => `api/clusters/workloads`,
```

Comment:
If we're going to continue adding the cluster routes and removing the namespace routes altogether then imo we should include cluster in the route instead of passing it as a query param e.g. `api/clusters/{clusterName}/workloads`. We also need to be sure that whatever is returning the list of clusters that the frontend iterates over to do these requests is stable and accurate and will always return at least something since it is a now a pre-req for doing any of these other requests. Basically this has ended up being a large change to the API and it's one that I support but it does affect the whole app.

Reply:
The reason of having 'clusterName' as a param is historical, from 'clusterNameFromQuery' method, where the `clusterName` param was not mandatory and had a default value. https://github.com/kiali/kiali/blob/master/handlers/utils.go#L173
But I agree that in future we need to switch to `api/clusters/{clusterName}/[workloads][apps][services]` requests.